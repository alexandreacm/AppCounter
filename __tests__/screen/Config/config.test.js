import 'react-native';
import React from 'react';
import {
  matchSnapshotWithTheme,
  renderWithTheme
} from '@/helpers/functions/testing';

import ConfigScreen from '@/screens/Config';

describe('tests in ConfigScreen', () => {
  test('matches snapshot', () => {
    matchSnapshotWithTheme(<ConfigScreen />);
  });

  test('renders correctly', () => {
    renderWithTheme(<ConfigScreen />);
  });
});
