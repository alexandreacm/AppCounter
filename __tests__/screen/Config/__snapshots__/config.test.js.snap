// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in ConfigScreen matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#0082c9",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 16,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 16,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#FFF",
          "fontSize": 20,
          "fontWeight": "bold",
          "textAlign": "left",
        },
      ]
    }
  >
    Counters
  </Text>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "marginBottom": 10,
          "marginTop": 10,
          "paddingBottom": 20,
          "paddingLeft": 20,
          "paddingRight": 20,
          "paddingTop": 20,
          "width": "100%",
        },
      ]
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#d8d8d8",
          "borderRadius": 10,
          "height": 60,
          "justifyContent": "center",
          "marginBottom": 10,
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
          "opacity": 1,
          "width": "50%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#14437b",
              "fontSize": 16,
              "fontWeight": "bold",
              "textAlign": "center",
            },
          ]
        }
      >
        Add Counter
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#d8d8d8",
          "borderRadius": 10,
          "height": 60,
          "justifyContent": "center",
          "marginBottom": 10,
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
          "opacity": 1,
          "width": "50%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#14437b",
              "fontSize": 16,
              "fontWeight": "bold",
              "textAlign": "center",
            },
          ]
        }
      >
        Remove Counter
      </Text>
    </View>
  </View>
  <Text
    style={
      Array [
        Object {
          "color": "#FFF",
          "fontSize": 16,
          "fontWeight": "bold",
          "textAlign": "center",
        },
      ]
    }
  >
    Increment / Decrement 
  </Text>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "borderColor": "#FFF",
          "borderStyle": "dotted",
          "borderWidth": 2,
          "flexDirection": "row",
          "height": 170,
          "justifyContent": "center",
          "marginBottom": 10,
          "paddingBottom": 25,
          "paddingLeft": 25,
          "paddingRight": 25,
          "paddingTop": 25,
          "top": 10,
          "width": "100%",
        },
      ]
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFF",
          "borderRadius": 10,
          "height": 60,
          "justifyContent": "center",
          "marginBottom": 10,
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
          "opacity": 1,
          "width": 60,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#2f3766",
              "fontSize": 30,
            },
            undefined,
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFF",
          "borderRadius": 10,
          "height": 60,
          "justifyContent": "center",
          "marginBottom": 10,
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
          "opacity": 1,
          "width": 60,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#2f3766",
              "fontSize": 30,
            },
            undefined,
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
